src/main/io/serial_cli.c

```diff
@@ -689,7 +689,9 @@ const clivalue_t valueTable[] = {
     { "d_vel",                      VAR_UINT8  | PROFILE_VALUE, .config.minmax = { PID_MIN,  PID_MAX } , PG_PID_PROFILE, offsetof(pidProfile_t, D8[PIDVEL])},
 
     { "yaw_p_limit",                VAR_UINT16 | PROFILE_VALUE, .config.minmax = { YAW_P_LIMIT_MIN, YAW_P_LIMIT_MAX } , PG_PID_PROFILE, offsetof(pidProfile_t, yaw_p_limit)},
-    { "dterm_cut_hz",               VAR_UINT16 | PROFILE_VALUE, .config.minmax = {0, 500 } , PG_PID_PROFILE, offsetof(pidProfile_t, dterm_cut_hz)},
+    { "dterm_lowpass_hz",           VAR_UINT16 | PROFILE_VALUE, .config.minmax = { 0, 500 } , PG_PID_PROFILE, offsetof(pidProfile_t, dterm_lpf_hz)},
+    { "dterm_nrd",                  VAR_UINT8  | PROFILE_VALUE, .config.minmax = { 0, PID_MAX_NRD } , PG_PID_PROFILE, offsetof(pidProfile_t, dterm_noise_robust_differentiator)},
```

Comment:
nrd is kind of hard to understand IMO. Why not dterm_ndifferentiator or something like that?